docs/markdown/Reference-tables.md

```diff
@@ -377,6 +377,26 @@ machine](#Environment-variables-per-machine) section for details.
 *The old environment variables are still supported, but are deprecated
 and will be removed in a future version of Meson.*
 
+*changed in 1.3.0* Paths with spaces were split unconditionally to extract
+components such as the [path to Ccache](Feature-autodetection.md#ccache),
+intrinsic compiler flags like `-m32` or `--target`, etc. This broke passing
+a hardcoded compiler path to CMake subprojects. To work around this, paths
+must be wrapped with double quotes:
+
+```bash
+export CC='"C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/cl.exe"'
+```
+
+You can also set the values through [machine files](Machine-files.md#binaries).
+
+*New in 1.3.0* Wrapping into an array is no longer needed when the path points
```

Comment:
Sorry. One more nit. The thing that's no longer necessary here is wrapping in double quotes, not wrapping in an array, right?

Reply:
Good catch, fixed.